airgun/utils.py

```diff
@@ -84,7 +84,7 @@ def retry_wrapper(*args, **kwargs):
         for i in range(attempts):
             try:
                 return method(*args, **kwargs)
-            except (Exception):
+            except (TimedOutError):
```

Comment:
Is it a TimeoutError (python in-built) or TimedOutError (Needs to be imported from where its originated , can you import it first?)

Reply:
It should be coming from `waitfor`